usrsctplib/user_socket.c

```diff
@@ -2406,6 +2422,16 @@ usrsctp_getsockopt(struct socket *so, int level, int option_name,
 				*option_len = (socklen_t)sizeof(struct linger);
 				return (0);
 			}
+		case SO_ERROR:
```

Comment:
I committed this with minor modifications separately, since it is not related to this PR.